src/bomsquad/vulndb/cli/ingest.py

```diff
@@ -15,11 +15,17 @@ def _nvd_ingest(
     offset: int = typer.Option(default=0, help="Offset into available entries to begin wtih"),
     update: bool = typer.Option(default=False, help="Acquire records newer than current data"),
 ) -> None:
-    db = NVDDB()
+    cp = Checkpoints()
     if scope == "cve" or scope is None:
-        Ingest.cve(offset, last_mod_start_date=db.cve_last_modified() if update else None)
```

Comment:
If we just handle this internal to Ingest.*, then it's entirely encapsulated within (and only needs to be tested behavior of) Ingest.